src/BowlingGame.js

```diff
@@ -0,0 +1,33 @@
+function Bowling(frame = new Frame()) {
+
+  this._currentFrame = frame;
+  this._result = 0;
+  this._frames = [];
+
+};
+
+Bowling.prototype.roll = function(pins) {
+  this._currentFrame.roll(pins);
+
+  if (this._frames.length !== 0) {
+
+    if (this._frames[this._frames.length - 1].isStrike === true) {
```

Comment:
Possibly reconsider either your names or this line of code as it can be confusing to outside devs to understand what's going on here.